R/na_util.R

```diff
@@ -85,3 +85,34 @@ impute_na <- function(target, type = mean, val = 0, ...) {
     })
   }
 }
+
+# same as zoo::na.locf, but fills only between non-NA values.
+# e.g. fill_between_v(c(NA,2,NA,3,NA)) returns c(NA, 2, 2, 3,NA).
```

Comment:
can you list all the options supported for direction?

Reply:
added comment.